src/Core/src/Handlers/Layout/LayoutHandler.Windows.cs

```diff
@@ -1,19 +1,25 @@
 using System;
 using Microsoft.Maui.Graphics;
 using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class LayoutHandler : ViewHandler<ILayout, LayoutPanel>
 	{
+		/// <summary>Children of the platform view.</summary>
+		/// <remarks>Stored for performance reasons.</remarks>
+		UIElementCollection? _platformViewChildren;
```

Comment:
I know this makes it better, but putting view specific data on handlers forces people to derived from the handler to customize sometimes.

What happens if you use an attached property on the platform view?

Reply:
I meant attached because I forgot we owned the type. But since it is our type, your PR is much better and we can even add the old children api to the banned apis